scripts/sysman/memoryBwCounterValidBits.yml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Bandwidth Counter Valid Bits"
+version: "1.8.1"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Memory Bandwidth Counter Valid Bits Extension Name"
+version: "1.8.1"
+name: $S_MEMORY_BW_COUNTER_VALID_BITS_EXP_NAME
+value: '"$XS_extension_mem_bw_counter_valid_bits"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Memory Bandwidth Counter Valid Bits Extension Version(s)"
+version: "1.8.1"
+name: $s_mem_bw_counter_bits_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties for reporting valid bit count for memory bandwidth counter value"
+version: "1.8.1"
+class: $sMemory
+name: $s_mem_bw_counter_bits_exp_t
+details:
+    - "Number of valid counter bits"
+members:
+    - type: $s_structure_type_t
+      name: stype
+      desc: "[in] type of this structure"
+    - type: "void*"
+      name: pNext
+      desc: "[in,out][optional] must be null or a pointer to an extension-specific structure (i.e. contains stype and pNext)."
+      init: nullptr
+    - type: uint32_t
+      name: memoryBwCounterValidBits
```

Comment:
Can we have bitCount, since the structure name already has memory bandwidth Counter ?

Reply:
changed to validBitsCount